src/app/docs/vertical-tabs/vertical-tabs-docs.component.html

```diff
@@ -0,0 +1,272 @@
+        [tabHeaderCount]="tab.tabHeaderCount" [disabled]="tab.disabled">
+        {{ tab.content }}
+      </sky-vertical-tab>
+    </sky-vertical-tabset>
+  </sky-docs-demo>
+
+  <sky-docs-demo>
+    <sky-docs-demo-control-panel (selectionChange)="onDemoSelectionChangeGroups($event)">
+      <sky-docs-demo-control-panel-section>
+        <sky-docs-demo-control-panel-checkbox label="Include counts" propertyName="counts" [checked]="false">
+        </sky-docs-demo-control-panel-checkbox>
+      </sky-docs-demo-control-panel-section>
+    </sky-docs-demo-control-panel>
+
+    <sky-vertical-tabset showTabsText="Tab list">
+
+      <sky-vertical-tabset-group *ngFor="let group of groups" [groupHeading]="group.heading" [open]="group.isOpen"
+        [disabled]="group.isDisabled">
+
+        <sky-vertical-tab *ngFor="let tab of group.subTabs" [active]="tab.active" [tabHeading]="tab.tabHeading"
+          [tabHeaderCount]="tab.tabHeaderCount" [disabled]="tab.disabled">
+          {{ tab.content }}
+        </sky-vertical-tab>
+      </sky-vertical-tabset-group>
+    </sky-vertical-tabset>
+  </sky-docs-demo>
+
+  <sky-docs-design-guidelines>
+    <sky-docs-demo-page-section heading="Usage">
+      <sky-docs-design-guideline heading="Use when" headingStyle="success">
+        <p>
+          Use vertical tabs when the number of tabs and/or the tab label text is expected to exceed the horizontal space
+          available.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          caption="Do use vertical tabs when it will be common to have a large number of tabs." captionType="success"
+          imageSource="~/assets/img/guidelines/vertical-tabs/vertical-tabs-usage-1.png">
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline>
+        <p>
+          Use grouped vertical tabs when the tabs naturally fit into multiple categories.
+        </p>
+        <sky-docs-design-guideline-thumbnail caption="Do use grouped vertical tabs when the tabs can be categorized."
+          captionType="success" imageSource="~/assets/img/guidelines/vertical-tabs/vertical-tabs-usage-2.png">
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline heading="Don't use when" headingStyle="danger">
+        <p>
+          Don’t use vertical tabs when horizontal tabs are also suitable.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          caption="Don't use vertical tabs when there is enough space for horizontal tabs." captionType="danger"
+          imageSource="~/assets/img/guidelines/vertical-tabs/vertical-tabs-usage-3.png">
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline>
+        <p>
+          Don’t combine vertical tabs and horizontal tabs. Change the workflow instead of nesting multiple sets of tabs.
```

Comment:
Should we also have a "Don't use" guideline for not nesting vertical tabs within other vertical tabs? We have a guideline like that for horizontal tabs, and we hint at it here in the second sentence. It seems to me like we should have that as a distinct "Don't use" when item here.

Reply:
probably so, or maybe make the wording more generic to not next tabs in tabs (regardless of regular or vertical) and show both examples.

Reply:
I'll try to go the more-generic route in my PR into this branch.

Reply:
I updated this in my branch where I'm adding descriptions for the dev properties and also implementing edits to the design guidelines. @Blackbaud-AdamFunderburk and @Blackbaud-ToddRoberts, if you guys can take a the PR for those changes at https://github.com/blackbaud/skyux-tabs/pull/83, I'd appreciate it. No rush though as I'm still trying to resolve questions about some dev properties.